controllers/user_controller.go

```diff
@@ -0,0 +1,65 @@
+package controllers
+
+import (
+	"net/http"
+
+	"github.com/drift-org/backend/models"
+	"github.com/gin-gonic/gin"
+	"github.com/kamva/mgm/v3"
+)
+
+type UserController interface {
+	Info(context *gin.Context)
+}
+
+type userController struct {
+}
+
+func NewUserController() UserController {
+	return &userController{}
+}
+
+func (ctrl *userController) Info(context *gin.Context) {
+	// Id refers to MongoDB user object ID. Specificty refers to the verbocity of the returned friends array
+	// 0 returns ONLY friend user IDs, 1 returns friend user IDs AND usernames
+	type ICreate struct {
+		Id          string `json:"id"`
+		Specificity int    `json:"specificity"`
```

Comment:
For specificity, we should make the default 0 (so no need for required binding). BUT, make sure to document that above.